hcloud/schema/storage_box_subaccount.go

```diff
@@ -0,0 +1,64 @@
+package schema
+
+import "time"
+
+// StorageBoxSubaccount defines the schema of a Storage Box subaccount.
+type StorageBoxSubaccount struct {
+	ID             int64                               `json:"id"`
+	Username       string                              `json:"username"`
+	HomeDirectory  string                              `json:"home_directory"`
+	Server         string                              `json:"server"`
+	AccessSettings *StorageBoxSubaccountAccessSettings `json:"access_settings"`
+	Description    string                              `json:"description"`
+	Labels         map[string]string                   `json:"labels"`
+	Created        time.Time                           `json:"created"`
+	StorageBox     int64                               `json:"storage_box"`
+}
+
+// StorageBoxSubaccountAccessSettings defines the schema of a Storage Box subaccount's access settings.
+type StorageBoxSubaccountAccessSettings struct {
+	ReachableExternally *bool `json:"reachable_externally,omitempty"`
+	Readonly            *bool `json:"readonly,omitempty"`
+	SambaEnabled        *bool `json:"samba_enabled,omitempty"`
+	SSHEnabled          *bool `json:"ssh_enabled,omitempty"`
+	WebDAVEnabled       *bool `json:"webdav_enabled,omitempty"`
```

Comment:
These are all marked as required in the spec, so can be `bool` instead of `*bool`